src/detector/implementations/abstract_marker_interface.py

```diff
@@ -0,0 +1,22 @@
+import abc
+from src.common import \
+    EmptyResponse, \
+    ErrorResponse
+from src.detector.api import \
+    GetDetectionParametersResponse, \
+    GetMarkerSnapshotsResponse
+
+class AbstractMarkerInterface(abc.ABC):
+    # TODO: how much of this is aruco specific?
```

Comment:
Everything in this file looks ok to me. If anything, probably more could be moved to `AbstractMarkerInterface` (things like status)